src/javascript/app/pages/cashier/payment_agent_withdraw.js

```diff
@@ -197,8 +190,13 @@ const PaymentAgentWithdraw = (() => {
                 $('#successMessage').css('display', '')
                     .attr('class', 'success-msg')
                     .html($('<ul/>', { class: 'checked' }).append($('<li/>', { text: localize('Your request to withdraw [_1] [_2] from your account [_3] to Payment Agent [_4] account has been successfully processed.', [request.currency, getNumberFormat(request.amount, request.currency), Client.get('loginid'), agent_name]) })));
+                
+                // Set PA details.
+                $('#agentName').text(agent_name);
+                $('#agentWebsite a').attr('href', agent_website).text(agent_website);
+                $('#agentEmail a').attr('href', `mailto:${agent_email}`).text(agent_email);
+                $('#agentTelephone a').attr('href', `tel:${agent_telephone}`).text(agent_telephone);
```

Comment:
please use javascript instead of jquery

Reply:
The rest of the file uses jQuery, it would not be consistent. Why do we want to do this?

Reply:
because javascript is faster.